backend/internal/database/book.go

```diff
@@ -49,23 +51,51 @@ func (db *BookDatabase) FetchBookList(ctx context.Context) ([]models.Book, error
  * For use after a user has selected a book to read
  */
 func (db *BookDatabase) FetchBookDetailsByID(ctx context.Context,
```

Comment:
Minor thing but `ByID` isn't quite accurate anymore, we could do `ByIDAndLang` or we could just do `FetchBookDetails` - I think the latter is concise without affecting clarity, since you kinda always need the ID and lang to get book details.